apps/postgresql.go

```diff
@@ -130,13 +131,27 @@ func (p LoggingProcessorPostgresql) Components(ctx context.Context, tag string,
 						},
 					},
 				},
+				{
+					// This parser matches the default log_line_prefix from SLES12 in our testing
+					// log_line_prefix = '%m %d %u [%p]'
```

Comment:
Is there really no space at the end? Sigh!

Reply:
Yup! Confirmed.